.github/workflows/nodejs.yml

```diff
@@ -1,20 +1,19 @@
 name: Node CI
 
 on:
-  pull_request:
-    branches:
-      - master
+  pull_request: {}
   push:
-    branches:
-      - master
+    branches-ignore:
+      - trying.tmp
+      - staging.tmp
 
 jobs:
   build:
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x]
+        node-version: [8.x, 10.x, 12.x, 13.x]
```

Comment:
Node 8.x is EOL'ed starting with this year. We can remove it🎉

Reply:
Yep, and all code specifically targeting it.